Add Makefile for simpler installation
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,10 @@
+all:
+
+install:
+	install -D -m755 bin/run.sh $(DESTDIR)/usr/bin/hornetq-run
+	install -D -m755 bin/stop.sh $(DESTDIR)/usr/bin/hornetq-stop
+	mkdir -p $(DESTDIR)/usr/share/hornetq
+	cp -r lib/ $(DESTDIR)/usr/share/hornetq/
+	cp -r schemas/ $(DESTDIR)/usr/share/hornetq/
+	mkdir -p $(DESTDIR)/etc/hornetq
+	cp -r config/stand-alone/ $(DESTDIR)/etc/hornetq/
